cmds/create-ffs/create-ffs.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2018 the LinuxBoot Authors. All rights reserved
```

Comment:
2019

Reply:
I'll keep it 2019, I think we should set the copyright on all the files?